Gemfile.lock

```diff
@@ -1,74 +1,84 @@
 GEM
   remote: http://rubygems.org/
   specs:
-    actioncable (6.0.5.1)
-      actionpack (= 6.0.5.1)
+    actioncable (7.0.4)
```

Comment:
The `Gemfile.lock` here shows version 7 for Rails.

The reason is that `run_tests.sh` runs through multiple versions of Rails so that ITRG's team can easily test this dependency (in the future) against multiple Rails versions when doing upgrades.

Because the test script runner tested Rails 7.0 last, the Gemfile.lock is updated as such. However, for all intent and purposes, the Gemfile.lock should be ignored.